stable/nginx-ingress/values.yaml

```diff
@@ -9,6 +9,7 @@ controller:
     pullPolicy: IfNotPresent
     # www-data -> uid 33
     runAsUser: 33
+    allowPrivilegeEscalation: false
```

Comment:
This needs to be set to `true` when binding to ports 80 and 443 as a non-root user since the default image has the capabilities set directly on the binaries (see [Dockerfile](https://github.com/kubernetes/ingress-nginx/blob/nginx-0.25.0/rootfs/Dockerfile#L49-L53) for reference) and Kubernetes does not currently support ambient capabilities (see [kubernetes/kubernetes#56374](https://github.com/kubernetes/kubernetes/issues/56374) for reference).

Reply:
Done

Reply:
@taharah done